web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentList.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * FlaggedContentList class definition.
+ */
+
+namespace Drupal\dbcdk_community\Plugin\Block;
+
+use DBCDK\CommunityServices\Api\FlagApi;
+use DBCDK\CommunityServices\ApiClient;
+use Drupal\Core\Block\BlockBase;
+use Drupal\dbcdk_community\Content\FlaggableContent;
+use Drupal\dbcdk_community\Content\FlaggableContentRepository;
+
+/**
+ * Provides a block displaying a list of flagged content.
+ *
+ * @Block(
+ *   id = "flagged_content_list",
+ *   admin_label = @Translation("Flagged content list"),
+ * )
+ */
+class FlaggedContentList extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    /* var FlaggableContentRepository $flagged_content */
+    $flagged_content = \Drupal::service('dbcdk_community.content.flaggable_content_repository');
```

Comment:
There is nothing wrong with this but it is just MORE correct to use the static create() method and inject this as a dependency in the block constructor